Makefile

```diff
@@ -25,34 +28,58 @@ unlink:
 
 .PHONY: lint
 lint:
+	npm --prefix es-bs-base run lint
+# TODO: set up linting for es-ds-components and es-ds-docs
+# 	npm --prefix es-ds-components run lint
+# 	npm --prefix es-ds-docs run lint
+
+.PHONY: legacy-lint
+legacy-lint:
 	npx lerna run lint
 
 .PHONY: test
 test:
```

Comment:
Similar to above, I might suggest we make the `test` command test everything, whereas perhaps a `test-next` is scoped to newer/prime related packages.